pkgs/applications/editors/vim/plugins/neovim-require-check-hook.sh

```diff
@@ -10,7 +10,7 @@ neovimRequireCheckHook () {
 		# editorconfig-checker-disable
         export HOME="$TMPDIR"
         @nvimBinary@ -es --headless -n -u NONE -i NONE --clean -V1 \
-            --cmd "set rtp+=$out,${dependencies/ /,}" \
```

Comment:
hmm can you explain why we need to do this?

Reply:
see commit message: https://github.com/NixOS/nixpkgs/pull/339921/commits/03266aa56ad8bcc9d35c6f9d049af6d2e9d9db4a